entries/o/odoo.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Odoo": {
+    "domain": "odoo.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.odoo.com/documentation/latest/applications/general/auth/2fa.html",
+    "notes": "Email 2FA is available if enabled by administrator.",
```

Comment:
If email 2FA is available, I need proof of that. Either as a link to a documentation page or a screenshot of the setup/login process.
If it isn't available, please remove this element.